src/components/Download/DownloadOptionsForm/index.js

```diff
@@ -14,12 +14,15 @@ import { TransformationOptions } from './TransformationOptions'
 export const DownloadOptionsForm = ({
   dataset = null,
   buttonLabel = 'Download',
+  isProcessed = false,
   onSubmit = null
 }) => {
   const { push } = useRouter()
   const {
     dataset: datasetState,
+    createDataset,
     updateDataset,
+    regenratedDataset,
```

Comment:
```suggestion
    regeneratedDataset,
```

This happened in a few other places too, I would just do a quick sweep to make sure they all look correct.

Reply:
The typo was corrected 6b91dc91b4e5fd34d364d9bdcfb4ca6408fb6022.